light-client/src/predicates.rs

```diff
@@ -291,6 +319,8 @@ pub fn verify_overlap(
     Ok(())
 }
 
+/// Assuming the given light block is valid, check whether the voting
+/// power of its validators meet the required threshold.
 pub fn has_sufficient_voting_power(
```

Comment:
It looks like this function should replace `verify_commit_full` in the current implementation, however it checks the power of commit votes with respect to `trust_threshold`(> 1/3) and the validator set of the header. Which seems like it is not correct as the check we are trying to do is confirm that the consensus has been reached and this should be done with respect to a threshold of 2/3. The mentioned check for >2/3 is done in `has_sufficient_signers_overlap`.

Reply:
Same as above.